rails-questions.md

```diff
@@ -2,33 +2,52 @@
 
 1. What is the command to create a new Rails application with a Postgres database?
 
+rails new (appname) -d postgresql -T
```

Comment:
Totally! I'd want to also mention the use of PascalCase or snake_case for the app name.